### PR TITLE
[zh-cn] Resync the NPD task page

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
Resync `content/zh-cn/docs/tasks/debug/debug-cluster/monitor-node-health.md` according to https://github.com/kubernetes/website/pull/38520.

### CODE REVIEW

File: content/zh-cn/docs/tasks/debug/debug-cluster/monitor-node-health.md [hunk 1]
```diff
@@ -197,95 +192,125 @@ The Addon manager does not support `ConfigMap`.
...
 -->
-### 添加对其它日志格式的支持  {#support-other-log-format}
 
-内核监视器使用 
-[`Translator`](https://github.com/kubernetes/node-problem-detector/blob/v0.1/pkg/kernelmonitor/translator.go)
-插件转换内核日志的内部数据结构。
-你可以为新的日志格式实现新的转换器。
+## 导出器
+
+导出器向特定后端报告节点问题和/或指标。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/debug/debug-cluster/monitor-node-health.md:303]:
```suggestion
导出器（Exporter）向特定后端报告节点问题和/或指标。
```

File: content/zh-cn/docs/tasks/debug/debug-cluster/monitor-node-health.md [hunk 2]
```diff
@@ -17,18 +17,18 @@ weight: 20
...
 -->
 
 *节点问题检测器（Node Problem Detector）* 是一个守护程序，用于监视和报告节点的健康状况。
 你可以将节点问题探测器以 `DaemonSet` 或独立守护程序运行。
-节点问题检测器从各种守护进程收集节点问题，并以
-[NodeCondition](/zh-cn/docs/concepts/architecture/nodes/#condition) 和
-[Event](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#event-v1-core)
+节点问题检测器从各种守护进程收集节点问题，并以节点
+[Condition](/zh-cn/docs/concepts/architecture/nodes/#condition) 和
+[Event](/docs/reference/kubernetes-api/cluster-resources/event-v1)
```
Comment (twilight0620) [inline content/zh-cn/docs/tasks/debug/debug-cluster/monitor-node-health.md:31]: (/zh-cn/docs/reference/kubernetes-api/cluster-resources/event-v1)

File: content/zh-cn/docs/tasks/debug/debug-cluster/monitor-node-health.md [hunk 3]
```diff
@@ -197,99 +192,129 @@ The Addon manager does not support `ConfigMap`.
...
+  [kernel](https://github.com/kubernetes/node-problem-detector/blob/v0.8.12/config/kernel-monitor-counter.json),
+  [abrt](https://github.com/kubernetes/node-problem-detector/blob/v0.8.12/config/abrt-adaptor.json),
+  and [systemd](https://github.com/kubernetes/node-problem-detector/blob/v0.8.12/config/systemd-monitor-counter.json).
+-->
+- `SystemLogMonitor` 类型的守护程序根据预定义的规则监视系统日志并报告问题和指标。
+  你可以针对不同的日志源自定义配置如
+  [filelog](https://github.com/kubernetes/node-problem-detector/blob/v0.8.12/config/kernel-monitor-filelog.json)、
+  [kmsg](https://github.com/kubernetes/node-problem-detector/blob/v0.8.12/config/kernel-monitor.json)、
+  [kernel](https://github.com/kubernetes/node-problem-detector/blob/v0.8.12/config/kernel-monitor-counter.json)、
+  [abrt](https://github.com/kubernetes/node-problem-detector/blob/v0.8.12/config/abrt-adaptor.json)
```
Comment (twilight0620) [inline content/zh-cn/docs/tasks/debug/debug-cluster/monitor-node-health.md:223]: 每行前面多个空格，前面是、 不用再加空格了

**Pull request merged**